src/styles/mixins/_animation.scss

```diff
@@ -1,5 +1,26 @@
+@import '../constants/animation';
+
 @mixin bdl-transition($time-length, $properties...) {
-    transition-timing-function: cubic-bezier(0, 0, .6, 1);
+    transition-timing-function: $bdl-timingFunction;
     transition-duration: $time-length;
     transition-property: $properties;
 }
+
+@mixin bdl-animation($time-length) {
```

Comment:
Should this have a more specific name?

Reply:
yah, a name could better indicate what this is for. it standardizes the fade-in animation instead of the previous "skew and fade" animation when opening any overlays (menus, flyout headers, and any other overlays)